haskell/test/Test/Ligo/BaseDAO/Management.hs

```diff
@@ -191,24 +192,28 @@ notSetAdmin withOriginatedFn initialStorage =
   withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
     \[owner, wallet1] baseDao -> do
       withSender owner $ do
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner .! wallet1)
-        -- Make the call once again to make sure the admin still retains admin
-        -- privileges
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner .! wallet1)
+        inBatch $ do
+          call baseDao (Call @"Transfer_ownership")
+            (#newOwner .! wallet1)
+          -- Make the call once again to make sure the admin still retains admin
+          -- privileges
+          call baseDao (Call @"Transfer_ownership")
+            (#newOwner .! wallet1)
+          pure ()
 
 rewritePendingOwner
   :: MonadNettest caps base m
   => WithOriginateFn m -> WithStorage -> m ()
 rewritePendingOwner withOriginatedFn initialStorage =
   withOriginatedFn 3 (\(owner:_) -> initialStorage owner) $
     \[owner, wallet1, wallet2] baseDao -> do
-      withSender owner $ do
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner .! wallet1)
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner .! wallet2)
+      withSender owner $
+        inBatch $ do
```

Comment:
Nitpick, but I think it's unnecessary to indent this further, I suggest:
```suggestion
      withSender owner . inBatch $ do
```
(same in the other places).